binancial/data/get_klines_realtime.py

```
@@ -1,43 +1,66 @@
-
-    f = open(file_path, 'a')
-
-    stream.start()
-
-    def handle_socket_message(msg):
-        
-        f.flush()
-        
+    # Input validation
+    if not isinstance(symbol, str) or not symbol:
+        raise ValueError('Symbol must be a non-empty string')
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)